modules/binding-infra-node-workloads-using-taints-tolerations.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating-infrastructure-machinesets.adoc
+
+[id="binding-infra-node-workloads-using-taints-tolerations_{context}"]
+= Binding infrastructure node workloads using taints and tolerations
+
+If you have an infra node that has the `infra` and `worker` roles assigned, you
+must configure the node so that user workloads are not assigned to it.
+
+.Prerequisites
+
+* Configure additional MachineSets in your {product-title} cluster.
+
+.Procedure
+
+. Use the following command to add a taint to the infra node to prevent
+scheduling user workloads on it:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node-name> <key>:<effect>
+----
++
+For example:
++
+[source,terminal]
+----
+$ oc adm taint nodes node1 node-role.kubernetes.io/infra:NoSchedule
+----
++
+This example places a taint on `node1` that has key
```

Comment:
Much easier to just remove the worker label and have it be a purely infra node. This way it shouldn't get any workloads scheduled to workers.